src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -93,7 +93,9 @@ export default function * loadCityContent (
   // loadLanguages did not update the dataContainer this is needed. In case the previous call to fetchResourceCache
   // failed to download some resources an other call could fix this and download missing files.
   if (criterion.shouldRefreshResources() && !isCellularConnection) {
-    const resourceURLFinder = new ResourceURLFinder(allowedResourceHostNames)
+    const { cmsUrl, switchCmsUrl } = buildConfig()
+    const allowedHostNames = switchCmsUrl ? [cmsUrl, switchCmsUrl] : [cmsUrl]
```

Comment:
`cmsUrl` and `switchCmsUrl` are no hostnames!
Make sure `allowedHostNames` is the same as before for the integreat build.
Maybe we should just just add this attribute to the buildConfig.

Reply:
Added to build configs